core/code/search.js

```diff
@@ -25,6 +25,8 @@ addHook('search', function(query) {});
   selected or the search was cancelled by the user).
 */
 
+/* global L, addHook -- eslint */
```

Comment:
```suggestion
/* global L -- eslint */
```

`L` is a global type and should be specified here, but `addHook` is our function defined on the `window` object - we should reference it that way (use `window.addHook` instead)

Reply:
done